pkg/devfile/parser/data/2.0.0/devfileJsonSchema200.go

```diff
@@ -848,6 +848,14 @@ const JsonSchema200 = `{
 		  "name": {
 			"type": "string",
 			"description": "Optional devfile name"
+		  },
+		  "dockerfile": {
+			  "type":"string",
+			  "description": "Optional URL to remote Dockerfile"
+		  },
+		  "deployment-manifest":  {
+			  "type":"string",
+			  "description": "Optional URL to remote Deployment Manifest"
```

Comment:
Need to agree if we want to prefix with `.alpha`

Reply:
I thought we had to use alpha.dockerfile and alpha.deployment-manifest as that was what was agreed in the design?

Reply:
Yeah, need to update that @neeraj.laad

Reply:
These need to be `alpha.build-dockerfile` and `alpha.deployment-manifest`

Reply:
Done